db/migrate/20230714030342_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+
+      t.timestamps
```

Comment:
- [ ] @NalianAchei2020, great effort working on the changes requested by the previous reviewer, but I noticed you missed one of the required changes. the comment table should also be referenced from the user and post table.

e.g
```ruby
 create_table :comments do |t|
    t.text :text
    t.timestamps
    t.references :user, foreign_key: 
    t.references :post, foreign_key: true 
  end

Also migrate the database so that you can have your schema file update with the changes.